### ISSUE TITLE
Finding input for compute_vina_maps given a reference ligand

### ISSUE THREAD

**Issue by and-tos** [ccsb-scripps/AutoDock-Vina#369]
Is there a (python) tool in Vina to obtain the input for the `compute_vina_maps` method, i.e. center and boundaries of the docking box, given a reference ligand?

**Comment by diogomart** [comment]
Yes, in `mk_prepare_receptor.py` ([docs](https://meeko.readthedocs.io/en/release/rec_cli_options.html)), the resulting box will include all the atoms in the file specified with `--box_enveloping` with a minimum distance, specified by `--padding`, between any atom and any face of the docking box. For example:
```
mk_prepare_receptor.py -i dna.cif -o receptor -j -p --box_enveloping some_atoms.pdb --padding 8
```
